src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/ApplicationService.kt

```diff
@@ -70,6 +70,8 @@ class ApplicationService(
     pageCriteria: PageCriteria<String>,
   ): Pair<MutableList<Cas2ApplicationSummaryEntity>, PaginationMetadata?> {
     val response = if (prisonCode == null) {
+    val uid = user.id.toString()
+    val records = applicationSummaryRepository.findByUserId(uid, null)
```

Comment:
did you mean to check this in?

Reply:
Good catch - we were using it to debug. Removed now.